modules/electrophysiology_uploader/jsx/UploadForm.js

```diff
@@ -197,17 +206,20 @@ export default function UploadForm(props) {
         error = resp.error;
       }
     } else if (xhr.status == 0) {
-      error = 'Upload failed: a network error occured';
+      error = t('Upload failed: a network error occured',
+        {ns: 'electrophysiology_uploader'});
     } else if (xhr.status == 413) {
-      error = 'Please make sure files are not larger than '
-          + props.maxUploadSize;
+      error = t('Please make sure files are not larger than',
+        {ns: 'electrophysiology_uploader'})
+          + ' ' + props.maxUploadSize;
     } else {
-      error = 'Upload failed: received HTTP response code '
-          + xhr.status;
+      error = t('Upload failed: received HTTP response code',
+        {ns: 'electrophysiology_uploader'})
+          + ' ' + xhr.status;
     }
 
     swal.fire({
-      title: 'Submission error!',
+      title: t('Submission error!', {ns: 'electrophysiology_uploader'}),
       text: error,
```

Comment:
There are already one `Error:` key in the **loris.po** I think we should use this one instead of creating a new one. Thanks.